### PR TITLE
Make refactoring boilerplate more consistent.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @jasonmalinowski @petrroll @mavasani

File: src/EditorFeatures/Test/LinkedFiles/LinkedFileDiffMergingEditorTests.cs [hunk 1]
```diff
@@ -82,7 +82,7 @@ private class TestCodeRefactoringProvider : CodeRefactorings.CodeRefactoringProv
         {
             public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContext context)
             {
-                var document = context.Document;
+                var (document, span, cancellationToken) = context;
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Test/LinkedFiles/LinkedFileDiffMergingEditorTests.cs]:
```suggestion
                var document = context.Document;
```

File: src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs [hunk 2]
```diff
@@ -28,15 +28,13 @@ public UseExpressionBodyCodeRefactoringProvider()
...
         public override async Task ComputeRefactoringsAsync(CodeRefactoringContext context)
         {
-            if (context.Span.Length > 0)
+            var (document, textSpan, cancellationToken) = context;
+            if (textSpan.Length > 0)
             {
                 return;
             }
 
             var position = context.Span.Start;
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs]:
```suggestion
            var position = textSpan.Start;
```

File: src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs [hunk 3]
```diff
@@ -28,15 +28,13 @@ public UseExpressionBodyCodeRefactoringProvider()
...
+            if (textSpan.Length > 0)
             {
                 return;
             }
 
-            var position = context.Span.Start;
-            var document = context.Document;
-            var cancellationToken = context.CancellationToken;
-
+             var position = textSpan.Start;
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs]:
```suggestion
            var position = textSpan.Start;
```

File: src/Features/CSharp/Portable/UseExpressionBodyForLambda/UseExpressionBodyForLambdaCodeRefactoringProvider.cs [hunk 4]
```diff
@@ -23,15 +23,13 @@ public UseExpressionBodyForLambdaCodeRefactoringProvider()
...
         public override async Task ComputeRefactoringsAsync(CodeRefactoringContext context)
         {
-            if (context.Span.Length > 0)
+            var (document, textSpan, cancellationToken) = context;
+            if (textSpan.Length > 0)
             {
                 return;
             }
 
             var position = context.Span.Start;
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/UseExpressionBodyForLambda/UseExpressionBodyForLambdaCodeRefactoringProvider.cs]:
```suggestion
            var position = textSpan.Start;
```

**Comment by petrroll** [comment]
LGTM.

**Comment by CyrusNajmabadi** [comment]
@petrroll do you have commit privs? Or do we need another review?  Thanks!

**Comment by petrroll** [comment]
Looks good to me, as for another review, I actually don't know what the policy is.

**Comment by CyrusNajmabadi** [comment]
@mavasani @jinujoseph Do we need another review here?  Thanks!

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Pull request merged**